README.CentOS7.md

```diff
@@ -103,6 +103,31 @@ Customize and Verify your settings:
  * DocumentRoot should point to `/var/www/loris/htdocs`
  * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
 
+Modify the Apache configuration file for our production environment.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
+```
+
+**a.** Find & modify the `ServerName` to:
+```
+ServerName <production_server>
```

Comment:
not wise to use <> to indicate choice as it is used in tag.
A better choice would be something like `your.Loris.url.here:443`.   and port number 80 or 443

should be done at <VirtualHost> tag as done in Ubuntu by default loris install

Reply:
@PapillonMcGill thank you. Please see new commit for change.

Reply:
What if user do not use /var/www/loris as a base dir?
Still not showing as close as possible the specification in <baseLorisDir>/docs/config/apache2-site.

Reply:
There should be other examples in our docs of the `$yourURL` and  `$lorisroot` or `$basePath` type placeholder (i forget which) -- consistency with those is ideal.